srtcore/api.cpp

```diff
@@ -1813,7 +1817,7 @@ void CUDTUnited::updateMux(
    s->m_iMuxID = m.m_iID;
 
    HLOGF(mglog.Debug, 
-      "creating new multiplexer for port %i\n", m.m_iPort);
+      "creating new multiplexer for port '%i' and ip '%i'\n", m.m_iPort,m.m_piIP[0]);
```

Comment:
Printing IP address by printing a decimal version of the 4-byte integer containing the address is clumsy. Please use `SockaddrToString`. You might also want to translate it to `HLOGC` in order to use iostream-like syntax.